tests/views/test_path.py

```diff
@@ -19,7 +19,7 @@ def test_valid_repo_name_for_manifest(self):
 
         self.assertEqual(response.status_code, 302)
         self.assertTrue(response.headers['Content-Type'].startswith('text/html'))
-        self.assertTrue('foo/bar/manifests/2' in response.headers['Location'])
+        self.assertTrue('foo/bar/manifests/1' in response.headers['Location'])
```

Comment:
yes it is, it is enough here to check if the directory is points to schema 1 2, or lists.
Though this tests is failing here, it does pass on my env, i have no idea why it fails in travis, all seems to be good